examples/soap-calculator/package.json

```diff
@@ -43,14 +43,6 @@
     "access": "public"
   },
   "license": "MIT",
-  "files": [
-    "README.md",
-    "index.js",
-    "index.d.ts",
-    "dist*/src",
-    "dist*/index*",
-    "src"
-  ],
```

Comment:
Have we decided to remove the explicit listing of `files` in `package.json` now? IIRC, it was there to exclude certain files to be published. 

If so, we should clean up all packages.

Reply:
None of the other example packages have a `files` field.